server/public/model/job.go

```diff
@@ -102,6 +104,75 @@ func (j *Job) Auditable() map[string]interface{} {
 	}
 }
 
+func (j *Job) MarshalYAML() (any, error) {
+	return struct {
+		Id             string    `yaml:"id"`
+		Type           string    `yaml:"type"`
+		Priority       int64     `yaml:"priority"`
+		CreateAt       string    `yaml:"create_at"`
+		StartAt        string    `yaml:"start_at"`
+		LastActivityAt string    `yaml:"last_activity_at"`
+		Status         string    `yaml:"status"`
+		Progress       int64     `yaml:"progress"`
+		Data           StringMap `yaml:"data"`
+	}{
+		Id:             j.Id,
+		Type:           j.Type,
+		Priority:       j.Priority,
+		CreateAt:       timeutils.FormatMillis(j.CreateAt),
+		StartAt:        timeutils.FormatMillis(j.StartAt),
+		LastActivityAt: timeutils.FormatMillis(j.LastActivityAt),
```

Comment:
I know it's a lot of efford just to pretty print the time stamps. But I think it's worth the extra code as UNIX millis are hard to read